test/e2e/framework/framework.go

```diff
@@ -87,6 +84,13 @@ type Framework struct {
 	cleanupHandle CleanupActionHandle
 }
 
+var (
+	beforeSuiteFuncs []func()
```

Comment:
I guess the function signature might change in the future based on the usage, but right now it looks fine.

Reply:
The current uses don't need to pass or return anything and I can't imagine a case that would need to.